README.md

```diff
@@ -50,7 +50,7 @@ kotlin {
   sourceSets {
         commonMain { //   <---  name may vary on your project
             dependencies {
-                implementation "org.reduxkotlin:presenter-middlware:0.2.8"
+                implementation "org.reduxkotlin:presenter-middleware:0.2.8"
```

Comment:
I also noticed in your [Name Game app](https://github.com/patjackson52/NameGameKotlinMpp/blob/master/build.gradle#L49) that it is using a more updated version, wondering if it is worth updating the version number as well. Or is that release not ready?

Reply:
Thanks!  I will be looking at in a couple of weeks and updating all this libs.